Uchu.Core/Config/ConfigurationPathAttribute.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace Uchu.Core.Config
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public class ConfigurationPathAttribute : Attribute
+    {
+        /// <summary>
+        /// Returns an absolute path relative to a configuration path.
+        /// </summary>
+        /// <param name="configurationLocation">File location of the configuration.</param>
+        /// <param name="relativePath">Relative path to make absolute.</param>
+        /// <returns>Absolute path of the relative path.</returns>
+        public static string ReplaceRelativePath(string configurationLocation, string relativePath)
+        {
+            // Normalize the relative path.
+            configurationLocation ??= "";
+            relativePath ??= "";
+            relativePath = relativePath.Replace(configurationLocation.Contains('/', StringComparison.CurrentCulture) ? "\\" : "/", configurationLocation.Contains('/', StringComparison.CurrentCulture) ? "/" : "\\", StringComparison.CurrentCulture);
```

Comment:
Is this necessary, can't Windows also handle forward slashes?

Reply:
I think no, but I know that java provides a char of the os's path separators. Maybe c# has that as well?

Reply:
I am not sure if this is a requirement. In the past, I have gotten some weird behavior *not* normalizing paths, so this is more preventative. To confirm this, I would need to test without this on both Windows and Linux.

Reply:
On Linux you'll need to replace \\ with / but I think Windows can handle either. But if it works it works, no need to test everything again for a tiny change imo

Reply:
I *think* the initial reason I did this is because of `Path.Combine`. Having a mix of `\` and `/` may have caused issues for me in the past with *something*, but I don't remember what.